src/functions.ts

```diff
@@ -74,16 +74,26 @@ export interface CloudEventFunction<T = unknown> {
 export interface CloudEventFunctionWithCallback<T = unknown> {
   (cloudEvent: CloudEvent<T>, callback: Function): any;
 }
+
+/**
+ * A Typed function handler that may return a value or a promise.
+ */
+export interface TypedHandlerFunction<T = unknown, U = unknown> {
```

Comment:
is there an equivalent "TypedFunction" to the "HttpFunction"?

Reply:
i think what you're pointing out is that the naming could perhaps just be `TypedFunction` -- would that be better?